core/src/main/scala/akka/persistence/jdbc/testkit/internal/SchemaUtilsImpl.scala

```diff
@@ -0,0 +1,103 @@
+package akka.persistence.jdbc.testkit.internal
+
+import java.sql.Statement
+
+import scala.concurrent.Future
+
+import akka.Done
+import akka.actor.ClassicActorSystemProvider
+import akka.annotation.InternalApi
+import akka.dispatch.Dispatchers
+import akka.persistence.jdbc.db.SlickDatabase
+import akka.persistence.jdbc.db.SlickExtension
+import org.slf4j.Logger
+import slick.jdbc.H2Profile
+import slick.jdbc.JdbcBackend.Database
+import slick.jdbc.MySQLProfile
+import slick.jdbc.OracleProfile
+import slick.jdbc.PostgresProfile
+import slick.jdbc.SQLServerProfile
+
+/**
+ * INTERNAL API
+ */
+@InternalApi
+private[jdbc] object SchemaUtilsImpl {
+
+  def dropSchema(logger: Logger)(implicit actorSystem: ClassicActorSystemProvider): Future[Done] = {
+    val slickDb: SlickDatabase = loadSlickDatabase("jdbc-journal")
+    val (fileToLoad, separator) =
+      slickDb.profile match {
+        case PostgresProfile  => ("schema/postgres/postgres-drop-schema.sql", ";")
+        case MySQLProfile     => ("schema/mysql/mysql-drop-schema.sql", ";")
+        case OracleProfile    => ("schema/oracle/oracle-drop-schema.sql", "/")
+        case SQLServerProfile => ("schema/sqlserver/sqlserver-drop-schema.sql", ";")
+        case H2Profile        => ("schema/h2/h2-drop-schema.sql", ";")
+      }
+    applyScript(fromClasspathAsString(fileToLoad), separator, logger, slickDb)
+  }
+
+  def createSchema(logger: Logger)(implicit actorSystem: ClassicActorSystemProvider): Future[Done] = {
+    val slickDb: SlickDatabase = loadSlickDatabase("jdbc-journal")
+    val (fileToLoad, separator) =
+      slickDb.profile match {
+        case PostgresProfile  => ("schema/postgres/postgres-create-schema.sql", ";")
+        case MySQLProfile     => ("schema/mysql/mysql-create-schema.sql", ";")
+        case OracleProfile    => ("schema/oracle/oracle-create-schema.sql", "/")
+        case SQLServerProfile => ("schema/sqlserver/sqlserver-create-schema.sql", ";")
+        case H2Profile        => ("schema/h2/h2-create-schema.sql", ";")
+      }
+    applyScript(fromClasspathAsString(fileToLoad), separator, logger, slickDb)
+  }
+
+  def dropAndCreateSchema(logger: Logger)(implicit actorSystem: ClassicActorSystemProvider): Future[Done] =
+    dropSchema(logger).flatMap(_ => createSchema(logger))(actorSystem.classicSystem.dispatcher)
+
+  def applyScript(script: String, separator: String, configKey: String, logger: Logger)(
+      implicit actorSystem: ClassicActorSystemProvider): Future[Done] =
+    applyScript(script, separator, logger, loadSlickDatabase(configKey))
+
+  private def applyScript(schema: String, separator: String, logger: Logger, slickDb: SlickDatabase)(
```

Comment:
@ennru, at the end I did what you suggested. One can pass the whole script in a String and it gets applied. 

The thing is, if you want to load something from the classpath, better to read the file into a string and pass to this method. 

Note that the `applyScript` is the power-user variant, you are on your own to pass all the bits.